File: HB/pysrc/프로그래머스/레벨0/Day23/등수_매기기.py
def solution(score):
    answer = []
    answer_= []
    for i in range(len(score)):
        answer.append(score[i][0]+score[i][1])
    answer_ = answer.sort()
    print(answer_)
    return answer
solution([[80, 70], [90, 50], [40, 70], [50, 80]])

File: SB/src/프로그래머스/레벨0/Day25/등수_매기기.java
package 프로그래머스.레벨0.Day25;

import java.util.*;

public class 등수_매기기 {

    class Solution {
        public int[] solution(int[][] score) {
            List<Integer> scoreList = new LinkedList<>();

            for (int[] s : score) {
                scoreList.add((s[0] + s[1]));
            }
            scoreList.sort(Comparator.reverseOrder());

            int[] answer = new int[score.length];
            for (int i = 0; i < score.length; i++) {
                answer[i] = scoreList.indexOf(score[i][0] + score[i][1]) + 1;
            }
            return answer;
        }
    }
}
